manifests/b/Bosyun/BoardMix/1.3.8/Bosyun.BoardMix.locale.en-US.yaml

```diff
@@ -4,12 +4,12 @@
 PackageIdentifier: Bosyun.BoardMix
 PackageVersion: 1.3.8
 PackageLocale: en-US
-Publisher: BoardMix
+Publisher: boardmix
```

Comment:
```suggestion
Publisher: Bosyun
```

Reply:
I got this:
![image](https://user-images.githubusercontent.com/56779163/227097172-8d7acf31-964b-4546-8b00-f405ca9d3188.png)

Reply:
@stephengillie